.travis.yml

```diff
@@ -18,8 +18,8 @@ branches:
     - /^.*appveyor.*$/
 
 stages:
-  - check
-  - test
+#  - check
+#  - test
```

Comment:
Please remove this

Reply:
Ops, I though I did it on my fork only ...
Anyway I see valgrind still complains, so I'll set WIP in this PR and try to see where is the problem